docs_src/microservices/security/edgex-config-proxy.1.md

```diff
@@ -0,0 +1,127 @@
+% edgex-config-proxy(1) User Manuals edgex-config-proxy(1)
+
+# NAME
+
+**edgex-config-proxy** – Configure EdgeX proxy service
+
+# SYNOPSIS
+
+**edgex-config** proxy [OPTIONS] SUBCOMMAND [ARG...]
+
+# DESCRIPTION
+
+**edgex-config-proxy** configures the EdgeX proxy service.
+
+This command is used to configure the TLS certificate for external connections, create authentication tokens for inbound proxy access, and relevant utility functions.
+
+These commands require access to the secret store master key in order to generate temporary secret store access credentials.
+
+# OPTIONS
+
+  * **-m** _/path/to/resp-init.json_
+
+    Points to location of secret store master key. Defaults to /vault/config/assets/resp-init.json
+
+# COMMANDS
+
+  * **tls**
+
+    Configure inbound TLS certificate. This command will configure the TLS secrets into the secret store and re-deploy them to Kong. Requires additional arguments:
+
+    * --certificate _/path/to/certchain.pem_
+
+      Path to TLS leaf certificate (with optional appended certificate chain, not including a root certificate).
+
+    * --key _/path/to/private.pem_
+
+      Path to TLS private key.
+
+  * **adduser**
+
+    Create a proxy user using JWT public/private key authentication. Requires additional arguments:
+
+    * --jwt (required)
```

Comment:
If add user always creates user using JWT authentication plugin, why have this option at all and force use to always type this option.

Reply:
> If add user always creates user using JWT authentication plugin, why have this option at all and force use to always type this option.

Mostly because we haven't bottomed out on whether we want a DB-less Kong.  If we go that route, JWT is the only option.  If we decide to keep the DB and allow regular OAuth tokens, I just want to be able to make this optional, and have the use choose between implementations.